queries.sql

```diff
@@ -1,5 +1,3 @@
-/*Queries that provide answers to the questions from all projects.*/
-
 SELECT * FROM animals WHERE name LIKE '%mon%'
 SELECT name FROM animals WHERE date_of_birth  BETWEEN '2016-01-01' and '2019-12-31';
```

Comment:
- [ ] I can not find the screenshot to his query, we are expecting data from 2016-2019, i have looked at the screenshot and i'm unable find it
![Screenshot from 2023-05-01 19-29-28](https://user-images.githubusercontent.com/46388113/235515515-891fed7a-17f5-4573-8211-6d165b1ac988.png)